### PR TITLE
apps/examples/cxxtest/cxxtest_main.cxx: apply a coding style

### PR DESCRIPTION
**Pull request by Xenociel**
the opening brace at the beginning of the next line.
the brace at the 311 line of this document, will be wrote at next line.

before
	for (it = v3.begin(); it != v3.end(); ++it)
	{
after
	for (it = v3.begin(); it != v3.end(); ++it){
		cout << *it << ' ';
	}

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [278627898d7ec59fe7f42d6deaa96a6845026a1f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [278627898d7ec59fe7f42d6deaa96a6845026a1f] - Code Rule Check OK.

File: apps/examples/cxxtest/cxxtest_main.cxx [hunk 1]
```diff
@@ -308,8 +308,7 @@ static void test_vectors(void)
 	string words[4] = {"Hello", "World", "Good", "Luck"};
 	vector<string> v3(words, words + 4);
 	vector<string>::iterator it;
-	for (it = v3.begin(); it != v3.end(); ++it)
-	{
+	for (it = v3.begin(); it != v3.end(); ++it){
```
Comment (thapav) [inline apps/examples/cxxtest/cxxtest_main.cxx]: Please put space between for argument and brace as below:
for (it = v3.begin(); it != v3.end(); ++it) {

Comment (Xenociel) [reply apps/examples/cxxtest/cxxtest_main.cxx]: oh sorry i'll check it

**Comment by seinfra** [comment]
Target : [680f697866cf03b8ffba310d8167b4a4618e610d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [680f697866cf03b8ffba310d8167b4a4618e610d] - Code Rule Check OK.

**Comment by davidfather** [comment]
Would you please re-commit this PR on 'typo_codingrule' branch instead?
Please, refer to our revised contribution rules.

**Pull request closed without merging**